x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table.tsx

```diff
@@ -190,7 +190,7 @@ const getColumns = ({
         }}
       />
     ),
-    width: '30px',
+    width: '3%',
```

Comment:
percentage are subject to change, on my ultrawide monitor it would be too much. any reason not to use pixels here?

Reply:
ah kk, no reason regarding the not using pixels, I just thought to keep it consistent with all the other rows (all other rows are using percentage)